dops/src/modules/dgen/dto/request/create-generated-document.dto.ts

```diff
@@ -71,6 +71,7 @@ export class CreateGeneratedDocumentDto {
           lastName: 'Hamilton',
           phone1: '(778) 952-1234',
           email: 'lewis@f1.com',
+          additionalEmail: 'lewis2@f1.com',
```

Comment:
not a big deal, but since we already have a small convention of `phone1` and `phone2`, then perhaps `email` and `email2` would've kinda worked as well?

Reply:
tbh phone1 and phone2 are the one's that should be changed imo, since they number suffix doesn't convey any meaning, but at this point we could leave it for now

Reply:
yup - agreed there (on both) 👍